src/application/agora/hooks/index.ts

```diff
@@ -0,0 +1,2 @@
+export { useTwitterVisibleAgoraNodes } from './use-twitter-visible-agora-nodes';
```

Comment:
is `useTwitterVisibleAgoraNodes` used outside of hooks directory? If not let's remove re-export

Reply:
I've used it directly in the `ProposalMainContainer` component but forgot to remove the `useTwitterProposalsToDisplay` where it was used initially.